README.rst

```diff
@@ -903,6 +903,13 @@ List locations with devices deployed (in GeoJSON format)
 
     GET /api/v1/controller/location/geojson/
 
+You can filter using ``organization_slug`` to list location of
+devices from that organization
+
+.. code:: text
+
+    GET /api/v1/controller/location/geojson/?organization_slug=<organization_slug>
+
```

Comment:
I didn't add a new heading since this is only extension of the previous endpoint.